draft-ietf-spice-sd-cwt.md

```diff
@@ -281,13 +283,16 @@ First she requests an SD-CWT from her issuer. The issuer generates an SD-CWT as
     ],
     / swversion / 271 : [
       "3.5.5",
-      /redacted version "4.1.7" /
-      { "...":  h'a0f74264a8c97655c958aff3687f1390' +
-                h'ed0ab6f64cd78ba43c3fefee0de7b835' }
+      / redacted version "4.1.7" /
+      {
+        / redacted_claim_element / 14:
+        h'a0f74264a8c97655c958aff3687f1390' +
+        h'ed0ab6f64cd78ba43c3fefee0de7b835'
+      }
```

Comment:
This doesn't work. There is a reasonable chance that some array out in the world will have an array element which takes a map of an integer to something and already uses the integer key value 14. The "..." is weird and ugly because it is supposed to be so weird and so ugly that nobody in their right mind is using a map with a key of "...".

Reply:
we're gonna have keep processing this, here: https://github.com/ietf-wg-spice/draft-ietf-spice-sd-cwt/issues/19